astropy/coordinates/baseframe.py

```diff
@@ -1333,16 +1333,13 @@ def is_frame_attr_default(self, attrnm):
         return attrnm in self._attr_names_with_defaults
 
     @staticmethod
-    def _frameattr_equiv(left_fattr, right_fattr):
+    def _frameattr_equiv(left_fattr, right_fattr):  # noqa: PLR0911
         """
         Determine if two frame attributes are equivalent.  Implemented as a
         staticmethod mainly as a convenient location, although conceivable it
         might be desirable for subclasses to override this behavior.
 
-        Primary purpose is to check for equality of representations.  This
-        aspect can actually be simplified/removed now that representations have
-        equality defined.
```

Comment:
This remark was added in #10290 (authored by @ayshih and reviewed by @mhvk). I don't understand why it was considered better to write this down as an idea instead of actually doing it.

Reply:
I don't quite remember but looking back, it likely was a case of not letting perfect being the enemy of good enough - especially since it was a bugfix in an-about-to-be-released version...